A32NX/html_ui/Pages/A32NX_Utils/NXFMGCFlightPhases.js

```diff
@@ -0,0 +1,10 @@
+const FmgcFlightPhases = {
```

Comment:
Okay, sorry, to nitpick again on such a minor detail, but:
- I would usually expect an enum-ish object to have the singular name (`FlightPhase`)
- Fmgc is fine... however I'm personally not that strict with CamelCase so I'm personally also okay with FMGCFlightPhase, but maybe more people should chime in or voice their preference here.

Reply:
Yeah I personally went with camel case, but since FMGC is a name itself, we might want to stick with that.
I'd like to get peoples thoughts on this one.

Reply:
@tracernz thoughts?

Reply:
There are good arguments both ways re. FMGC vs. Fmgc and I don't really mind either. I think having FMGC in the name is a good move to avoid confusion with FWC or any other systems. It's easier to spot issue in future code review, e.g. if you see FmgcFlightPhase in some ECAM code you'd immediately be suspicious, whereas FlightPhase would seem normal on first inspection.